AsyncDisplayKit/ASCellNode.m

```diff
@@ -31,13 +31,13 @@ - (instancetype)init
   return self;
 }
 
-- (instancetype)initWithLayerBlock:(ASDisplayNodeLayerBlock)viewBlock
+- (instancetype)initWithLayerBlock:(ASDisplayNodeLayerBlock)viewBlock layerDidLoadBlock:(ASDisplayNodeLayerLoadedBlock)layerLoadedBlock
```

Comment:
I'm excited to have this feature!  API simplification request:

- Can we name both of these new arguments didLoadBlock: ?
- Can the block type be the same, ASDisplayNodeLoadedBlock or ASDisplayNodeDidLoadBlock? (latter is probably better..)

The argument to the block should simply be the node itself.  It's trivially easy to access the view and layer on the node, but fewer people know how to access the node from the view or layer, so I think it will provide possible utility in addition to the API simplicity.  I agree with your original design in that it is probably more convenient for the common case, but think it is important to minimize duplication between view & layer-backed APIs (as one of the primary values of layer-backing is its seamless compatibility when enabled on previously view-backed nodes)